src/arch/xtensa/configs/override/renoir_chrome_dts.config

```diff
@@ -0,0 +1,12 @@
+CONFIG_RENOIR=y
+CONFIG_TRACE_CHANNEL=7
+CONFIG_WRAP_ACTUAL_POSITION=y
+CONFIG_AMD_BT=n
+CONFIG_CORE_COUNT=1
+CONFIG_LP_MEMORY_BANKS=0
+CONFIG_HP_MEMORY_BANKS=10
+CONFIG_FORMAT_CONVERT_HIFI3=n
+CONFIG_LP_SRAM=n
+CONFIG_HAVE_AGENT=n
```

Comment:
The configs which are already in src/arch/xtensa/configs/renoir_defconfig doesn't need to be in the override config (line1 to line10). Unless the config value is different.

Reply:
Remove the duplicated lines in the override config.